src/test/java/janggi/piece/ChaTest.java

```diff
@@ -0,0 +1,168 @@
+package janggi.piece;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import janggi.position.Position;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class ChaTest {
+
+    @DisplayName("차은 위치 정보를 가진다,")
```

Comment:
```suggestion
    @DisplayName("차는 위치 정보를 가진다.")
```

Test DisplayName도 일관성이 있으면 좋습니다.

기물명으로 시작하는 경우도 있고 아닌 경우도 있네요.